PROTOCOL.md

```diff
@@ -757,19 +758,22 @@ Deletion Vectors are basically sets of row indexes, that is 64-bit integers that
 
-Since RoaringBitmap only covers 32-bit integers, we extend the format in a simple manner by keeping an array of 32-bit RoaringBitmaps and using the upper 32-bits to index into the array.
+The above description only applies to 32-bit bitmaps, but Deletion Vectors use 64-bit integers. In order to extend coverage from 32 to 64 bits, RoaringBitmaps defines a "portable" serialization format in the [RoaringBitmaps Specification](https://github.com/RoaringBitmap/RoaringFormatSpec#extention-for-64-bit-implementations). This format essentially splits the space into an outer part with the upper 32-bit "keys" indexing the lower 32-bit RoaringBitmaps.
+We use this format with two additions:
 
-The serialization format for such a `RoaringBitmapArray` is as follows (all numerical values are written in little endian byte order):
+1. We prepend a "magic number", which can be used to make sure we are reading the correct format and also retains the ability to evolve the format in the future.
+2. We require that every "key" (s. above) in the bitmap has a 0 as its highest order bit. This ensures that in Java, where values are read signed, we never read negative keys. 
+
+The concrete serialization format is as follows (all numerical values are written in little endian byte order):
 
 Bytes | Name | Description
 -|-|-
-0 — 3 | magicNumber | 1681511376; Indicates that the following bytes are serialised in this exact format. Future alternative—but related—formats must have a different magic number, for example by incrementing this one.
-4 — 7 | numBitmaps | The number of 32-bit bitmaps in the array
-`repeat for i in 0 to length` | | For each 32-bit RoaringBitmap
-`<start of i>` — `<start of i> + 3` | bitmapDataSize | Number of bytes of this bitmap’s serialized representation.
-`<start of i> + 4` — `<start of i> + 4 +  bitmapDataSize` | bitmapData | Serialized bytes in the RoaringBitmap standard serialization format.
+0 — 3 | magicNumber | 1681511377; Indicates that the following bytes are serialized in this exact format. Future alternative—but related—formats must have a different magic number, for example by incrementing this one.
+4 — end | bitmap | A serialized 64-bit bitmap in the portable standard format as defined in the [RoaringBitmaps Specification](https://github.com/RoaringBitmap/RoaringFormatSpec#extention-for-64-bit-implementations). This can be treated as a black box by any Delta implementation that has a native, standard-compliant RoaringBitmap library available to pass these bytes to.
+
+For a summary on RoaringBitmap's serialization format see [Roaring Bitmap Format Summary](#roaring-bitmap-format-summary).
 
 The format for storing DVs in file storage is one (or more) of these `RoaringBitmapArray`s per file, together with a checksum for each DV:
```

Comment:
This does not make sense now since `RoaringBitmapArray` is not defined earlier any more.